### PR TITLE
PAYEE-305: Upgraded to new gds library

### PR DESCRIPTION
**Pull request by Jamie22Butler**
- [x] Unit tests passing
- [x] Coverage reached
- [ ] Acceptance tests passing
- [x]  Browser Accessibility checks
- [ ] Reviewed

### CODE REVIEW

File: app/views/PotentialUnderpaymentView.scala.html [hunk 1]
```diff
@@ -20,62 +20,72 @@
...
+        preHeadingText = Messages("tai.iya.tax.you.owe.preHeading"),
+        displayBackLink = false,
+        showGdsVersion = true)
 }
 
-@mainTemplate(
-    title = viewModel.pageTitle,
-    articleClasses = Some("full-width"),
+@main(
+    title = messages(viewModel.pageTitle),
```
Comment (brendanstraughan) [inline app/views/PotentialUnderpaymentView.scala.html:43]: why is this wrapped in messages()?

File: app/views/PotentialUnderpaymentView.scala.html [hunk 2]
```diff
@@ -20,62 +20,72 @@
...
     authedUser = Some(user),
-    pageHeader = Some(header)
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    backLinkUrl = Some(routes.TaxAccountSummaryController.onPageLoad.url),
```
Comment (brendanstraughan) [inline app/views/PotentialUnderpaymentView.scala.html:46]: was this checked with UX?

File: test/views/html/PotentialUnderpaymentViewSpec.scala [hunk 3]
```diff
@@ -85,7 +85,7 @@ class PotentialUnderpaymentViewSpec extends TaiViewSpec {
       def cyOnlyDoc = document(cyOnlyViewModel)
 
       "display amount owing" in {
-        cyOnlyDoc must haveParagraphWithText(withPoundPrefix(MoneyPounds(cyOnlyViewModel.iyaCYAmount, 2)))
+        cyOnlyDoc must haveH2HeadingWithText(withPoundPrefix(MoneyPounds(cyOnlyViewModel.iyaCYAmount, 2)))
```
Comment (brendanstraughan) [inline test/views/html/PotentialUnderpaymentViewSpec.scala:88]: How come this has been changed to H2?

File: test/views/html/PotentialUnderpaymentViewSpec.scala [hunk 4]
```diff
@@ -106,7 +106,7 @@ class PotentialUnderpaymentViewSpec extends TaiViewSpec {
       def cyAndCYPlusOneDoc = document(cyAndCYPlusOneViewModel)
 
       "display amount owing" in {
-        cyAndCYPlusOneDoc must haveParagraphWithText(
+        cyAndCYPlusOneDoc must haveH2HeadingWithText(
```
Comment (brendanstraughan) [inline test/views/html/PotentialUnderpaymentViewSpec.scala:109]: same^?

Comment (Jamie22Butler) [reply test/views/html/PotentialUnderpaymentViewSpec.scala:109]: Same as the reason above :)

File: app/views/PotentialUnderpaymentView.scala.html (hunk 1 above)
Comment (Jamie22Butler) [reply app/views/PotentialUnderpaymentView.scala.html:43]: I was following the pattern from my previous tickets. I can change this if this isn't standard.

Comment (brendanstraughan) [reply app/views/PotentialUnderpaymentView.scala.html:43]: does it work? I would of thought this looks for a message key?

File: test/views/html/PotentialUnderpaymentViewSpec.scala (hunk 3 above)
Comment (brendanstraughan) [reply test/views/html/PotentialUnderpaymentViewSpec.scala:88]: cool, and it looks the same?

File: app/views/PotentialUnderpaymentView.scala.html (hunk 1 above)
Comment (Jamie22Butler) [reply app/views/PotentialUnderpaymentView.scala.html:43]: It works, no tests surrounding the title seems to fail, and this is used in the DOM correctly too.

**Review by sergi-hmrc — APPROVED** [review]

**Pull request merged**